APsystemsEZ1/__init__.py

```diff
@@ -234,9 +234,10 @@ async def get_device_power_status(self) -> Status:
 
         :return: 0/normal when on, 1/alarm when off
         """
-        return Status(int((await self._request("getOnOff"))["data"]["status"]))
+        response = await self._request("getOnOff")
+        return Status(response["data"]["status"]) if response else None
```

Comment:
Isn't here a int-mapping missing? So that the enum works?

Reply:
fixed